### PR TITLE
Upgrade to python3

### PR DESCRIPTION
**Pull request by tboerger**
Fixes https://github.com/drone-plugins/drone-ansible/issues/23

### CODE REVIEW

File: plugin.go [hunk 1]
```diff
@@ -173,20 +173,25 @@ func (p *Plugin) vaultPass() error {
 }
 
 func (p *Plugin) playbooks() error {
-	var playbooks []string
+	var (
```
Comment (lafriks) [inline plugin.go]: why expand this for single local variable definition?

Comment (tboerger) [reply plugin.go]: Because it's done with that schema on all plugins.

Comment (lafriks) [reply plugin.go]: For global variables yes but at least from drone code func local variables are defined as `var xxx ..`

Comment (tboerger) [reply plugin.go]: Not within the plugin org, I have written that like it on every plugin that I have authored.

Comment (lafriks) [reply plugin.go]: oh, ok, I was just asking ;)

**Pull request merged**